pkgs/top-level/all-packages.nix

```diff
@@ -5588,6 +5588,10 @@ let
 
   node_webkit = node_webkit_0_9;
 
+  nwjs_0_12 = callPackage ../development/tools/node-webkit/nw12.nix {
```

Comment:
@MasseGuillaume The sources still differ in SHA256. The installPhase differs between 0.12 and 0.11/0.9. That said, `nwjs-env`, `bits` and part of `installPhase` could be generalized. I can do this in a separate PR.